setup.json

```diff
@@ -31,32 +31,27 @@
     ],
     "reentry_register": true,
     "install_requires": [
-        "aiida-core >=1.0.0,<2.0.0",
-        "pycifrw >= 4.2",
-        "numpy < 1.17",
-        "pymatgen <= 2018.12.12",
-        "monty==2.0.4",
+        "aiida-core~=1.0",
+        "pycifrw~=4.2",
+        "pymatgen",
+        "monty~=2.0.4",
         "voluptuous",
         "six"
     ],
     "extras_require": {
         "testing": [
-            "pgtest==1.2.0",
+            "pgtest~=1.2.0",
             "pytest>=4.4,<5.0.0",
             "pytest-cov>=2.6.1,<3.0.0",
             "coverage",
-            "ase; python_version>='3.0'",
-            "ase<3.18; python_version<'3.0'"
+            "ase~=3.18"
```

Comment:
before for python3 it was just 'ase'. Do you still want to add this constraint?

Reply:
you're right - I don't need 3.18 if that's an issue
I've changed it to  ~=3.15